proposals/0281-visible-forall.rst

```diff
@@ -625,57 +625,10 @@ Syntax
    This change applies to ``∀`` (the ``UnicodeSyntax`` rendition of ``forall``)
    as well.
 
-4. Introduce a new extension, ``ListTupleTypeSyntax``, on by default,
```

Comment:
Could you also amend the “Corner Cases” section that still talks about `TupleOf2`?